tests/utils.go

```diff
@@ -3473,6 +3473,20 @@ func SkipIfPrometheusRuleIsNotEnabled(virtClient kubecli.KubevirtClient) {
 	}
 }
 
+func SkipIfSingleReplica(virtClient kubecli.KubevirtClient) {
```

Comment:
I think `tests/framework/checks/skips.go` would be a good place for these functions.

Reply:
Added a commit moving all the skips I could find in there to the `checks` package